python/pyarrow/table.pxi

```diff
@@ -1966,9 +2085,12 @@ cdef class Table(_PandasConvertible):
 
         return pyarrow_wrap_table(c_table)
 
-    def to_batches(self, max_chunksize=None, **kwargs):
+    def to_batches(self, max_chunksize=None):
         """
-        Convert Table to list of (contiguous) RecordBatch objects.
+        Convert Table to a list of RecordBatch objects.
+
+        Note that this method is zero-copy, it merely exposes the same data
+        under a different API.
```

Comment:
This is not necessarily true, as the chunking of each column does not have to be the same? (in which case you will get some copies?)

Reply:
Actually, I suppose we still only _slice_ data in that case, so still being zero-copy?

Reply:
Yes, it's still zero-copy thanks to slicing.